ci/jobs/deploy_commit_hooks.yml.j2

```diff
@@ -3,16 +3,14 @@ jobs:
     max_in_flight: 1
     plan:
       - in_parallel:
-          - get: dataworks-github-config
+          - get: dataworks-github-config-webhooks
```

Comment:
they aren't webhooks, so this is a bit confusing

Reply:
Agreed.  Fixed.